src/main/java/com/epam/controller/ToursController.java

```diff
@@ -37,37 +46,38 @@ public ModelAndView searchTours(@RequestParam String country, @RequestParam Stri
         ModelAndView toursModel = new ModelAndView();
         toursModel.setViewName("homepage");
         try {
-            LocalDate addStartDate = Validator.getDate(startDate,true);
-            LocalDate addEndDate = Validator.getDate(endDate,true);
-            toursModel.addObject("list",toursOfferService.searchTours(null,addStartDate, addEndDate));
+            LocalDate addStartDate = Validator.getDate(startDate, true);
```

Comment:
I see that your Inellij want to fix all our code style errors, but please in another pull request